docs/guides/configuration-guide/loadbalancer.md

```diff
@@ -25,42 +29,76 @@ hosts_additional_entries:
 
 ## TLS certificates
 
-To enable external TLS encryption:
+:::warning
+To avoid unnecessary additional work and problems, it is recommended that you configure TLS with the intended target
+configuration of the specific environment during the initial rollout process.
+:::
 
-```yaml title="environments/kolla/configuration.yml"
-kolla_enable_tls_external: "yes"
-```
+For example: Changes to the configuration of TLS or the fully qualified domain names (FQDNs) will result in new URLs (with and without the https prefix)
+stored in the Openstack databases, and in the case of self-signed certificates, the CA certificate must be distributed to all participating clients.
 
-To enable internal TLS encryption:
+As a result, the involved Ansible Plays must at least be executed in the correct order, and not all Ansible Plays can handle all possible configuration transitions on their own.
+Therefore, in some cases, manual adjustments must be made to the systems.
 
-```yaml title="environments/kolla/configuration.yml"
-kolla_enable_tls_internal: "yes"
-```
+For this reason, we recommend that you define domains, obtain certificates, and perform configurations in advance.
```

Comment:
I'm not sure I understand this either. "in advance", before doing what next?

Please be specific about the order of things being done, e.g. make a numbered list, if it is deemed important.

e.g. when you use let's encrypt, it's not possible to obtain certificates in advance.

You certainly need to define your used FQDN (domain) before deploying osism, but that's orthogonal to using TLS? What kind of "configuration" are you referring to here?